mmedit/engine/hooks/ema.py

```diff
@@ -69,22 +69,25 @@ def __init__(self,
             getattr(self, interp_mode), **self.interp_cfg)
 
     @staticmethod
-    def lerp(a, b, momentum=0.999, momentum_nontrainable=0., trainable=True):
-        """This is the function to perform linear interpolation between a and
-        b.
+    def lerp(a, b, momentum=0.001, momentum_nontrainable=1., trainable=True):
+        """Does a linear interpolation of two parameters/ buffers.
 
         Args:
-            a (float): number a
-            b (float): bumber b
-            momentum (float, optional): momentum. Defaults to 0.999.
-            momentum_nontrainable (float, optional): Defaults to 0.
-            trainable (bool, optional): trainable flag. Defaults to True.
-
+            a (torch.Tensor): Interpolation start point, refer to orig state.
+            b (torch.Tensor): Interpolation end point, refer to ema state.
+            momentum (float, optional): The weight for the interpolation
+                formula. Defaults to 0.001.
+            momentum_nontrainable (float, optional): The weight for the
+                interpolation formula used for nontrainable parameters.
+                Defaults to 1..
+            trainable (bool, optional): Whether input parameters is trainable.
+                If set to False, momentum_nontrainable will be used.
+                Defaults to True.
         Returns:
-            _type_: _description_
+            torch.Tensor: Interpolation result.
         """
         m = momentum if trainable else momentum_nontrainable
-        return a + (b - a) * m
+        return b + (a - b) * m
```

Comment:
Since momentum is a large value before, so we need to dd a warning, refer to https://github.com/open-mmlab/mmengine/blob/bf369da22ada3336e8f44e23c7de3f3117b9c84a/mmengine/model/averaged_model.py#L176

Reply:
It seems we times the original weight (`a`) with a small value. Is this right? 🤔